test/controllers/presenter.controller.spec.ts

```diff
@@ -255,32 +284,39 @@ describe("PRESENTER controller", () => {
 
   describe("POST with url params tests", () => {
     test(`redirect to the ${WHO_IS_MAKING_FILING_PAGE} page after a successful post from presenter page with url params`, async () => {
-      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_ROE_UPDATE
+      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_REDIS_REMOVAL
```

Comment:
This was always wrong - a copy/paste error when we mobbed it I think. Correcting.